docs/development.md

```diff
@@ -45,7 +45,7 @@ Done! Enjoy building your Probot app!
 
 The most important files note here are `index.js`, which is where the code for your app will go, and `package.json`, which makes this a standard [npm module](https://docs.npmjs.com/files/package.json).
 
-## Configure a GitHub App
+## Configuring a GitHub app
```

Comment:
I don't have strong feels about this, but "GitHub App" is a product name that is typically capitalized together, in the same way "GitHub Issues" is a name that is typically capitalized when used together. There may be other uses of "GitHub App" that we should update if we update it here.

Reply:
Good call. Updated.